custom-migrations/auth/V20__ADD_SAVE_DATASET_ROLE.sql

```diff
@@ -0,0 +1,60 @@
+use auth;
+
+SET @uuidRule = REPLACE(UUID(),'-','');
+INSERT INTO access_rule (
```

Comment:
This formatting is kinda gross. I respect matching columns to their values, but I think you can do more than 1 per line